utils/sub8_diagnostics/scripts/thruster_spinner.py

```diff
@@ -31,7 +31,10 @@ def ports_from_layout(layout):
             for name in thruster_names:
                 port_dict[name] = thruster_port
 
-            msg += "\n\tMotor id's on port: {}".format(thruster_port.motor_ids_on_port)
+            msg += "\n\tMotor id's on port: {}{}{}".format(
```

Comment:
Assuming you have a printer defined (`p = text_effects.Printer()`), this becomes `msg += str(p.text("Motor id's...").set_bold.blue(np.sort(...)))`
Could be even easier if msg is already a Printer type.